### ISSUE TITLE
Site Editor: Consider renaming Appearance > Editor

### ISSUE THREAD

**Issue by annezazu** [WordPress/gutenberg#43926]
## What problem does this address?

As the Site Editor has evolved and is going to continue to, including timing when to remove the beta label https://github.com/WordPress/gutenberg/issues/39293 and exploring how to have a better entry point https://github.com/WordPress/gutenberg/issues/36667#issuecomment-1228468407, questions have come up around whether the current labeling still makes sense to use. This issue seeks to gather ideas and feedback to better influence a possible rename. 

<img width="332" alt="Screen Shot 2022-09-06 at 6 20 22 PM" src="https://user-images.githubusercontent.com/26996883/188767820-de582d87-4861-4b1b-8218-edd097573cc7.png">

IMO this change should be done in alignment with removing the beta label to offer a more finalized option. 

## What is your proposed solution?

Thus far, I've seen the following thrown out:

- Customize
- Design
- Style

**Comment by carolinan** [comment]
I don't see a consensus or resolution here: https://make.wordpress.org/core/2022/07/27/giving-fse-a-more-user-friendly-name/

Both the link text and the title of the editor view is "Editor". I did not understand what is proposed to change - it doesn't say "Site editor".

**Comment by annezazu** [comment]
Yes! That discussion ties in as well which is why I put "site editor" in quotes as I'm trying to name the specific feature rather than just saying "Editor". Perhaps that's confusing though based on your reaction. Updated the title in case that helps :) Let me know if it doesn't. 

To be clear, this is an issue about renaming the label under Appearance that opens what is currently thought of as the Site Editor.

**Comment by autumnfjeld** [comment]
Won't the Site Editor be used for editing content as well?  In which case wouldn't that rule out terms like "Design" and "Style" which don't encompass the broader usage of the Site Editor?

Or is this menu "special" and meant to provide a specific UX even though the link leads to the same Site Editor where user's do all sorts of editing, not just style or design.

**Comment by jasmussen** [comment]
> Won't the Site Editor be used for editing content as well? In which case wouldn't that rule out terms like "Design" and "Style" which don't encompass the broader usage of the Site Editor?

That would depend on how you flow into it. At the moment you can move into template editing from the page editor, and back. This flow needs further iteration, but is an example of a seamless shifting of context that I could see working for the cases where you edit content in the site editor. 

Navigation-wise, however, I wouldn't think site editor should become a starting point for writing pages and posts. In that light, I could still see labels that that describe the main use case of the section.

**Comment by annezazu** [comment]
Noting that currently some [designs are showing as "Appearance > Design".](https://github.com/WordPress/gutenberg/pull/44770)

**Comment by scruffian** [comment]
Worked with a user today who did not expect the Site Editor to be in Appearance at all. I would suggest at the very least renaming to "Site Editor" and ideally moving to a top level item.

**Comment by carolinan** [comment]
I think renaming this again at this point will cause more confusion than clarity. Users, -beginners and advanced users and developer alike, are already expressing in forums and social media that all the UI changes are difficult to keep up with.

It may also be easier to guide single users than update all the documentation including screenshots. This does not only affect the documentation on WordPress.org but all third party articles...

**Comment by carolinan** [comment]
If needed, it can be part of the larger admin redesign but does not need to be addressed before that in my **_opinion.,_**

**Comment by carolinan** [comment]
If the menu item is difficult to find, perhaps the highlight on the dashboard is not working well either. I guess people don't often visit the dashboard.

**Comment by annezazu** [comment]
Noting that in exposing some various tooling related to Styles to Classic themes, [the "Design" option came up again as the path for an Appearance item.](https://github.com/WordPress/gutenberg/pull/66851) If we're able to have this be a pathway for Classic themes for exposing more gradual adoption options, it might be a natural next step for Block themes too.
